saw-remote-api/scripts/run_rpc_tests.sh

```diff
@@ -13,19 +13,9 @@ function run_test {
 echo "Setting up python environment for remote server clients..."
 poetry install
 
-# Ask cabal where the server is, and if that does work... check
```

Comment:
@pnwamk I've normalized these files to simply depend on `PATH`, so that it works well for both local development (with `cabal v2-exec`, i.e. `cabal v2-exec ./saw-remote-api/scripts/run_rpc_tests.sh`) and CI (running the test binary directly with specific `PATH`). Is this sufficient to meet your needs?